Work readiness.postman_collection.json

```diff
@@ -0,0 +1,626 @@
+{
+	"info": {
+		"_postman_id": "0aac0a05-b978-4589-bab2-6fdc0ed2824e",
+		"name": "Work readiness",
+		"schema": "https://schema.getpostman.com/json/collection/v2.0.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "signin",
+			"request": {
+				"auth": {
+					"type": "basic",
+					"basic": {
+						"password": "v6oP++I2dUO3.z,F2ahKcUM,8o<oGSnV(8ylu,C)SiOF7img!&kIw9a.(noH",
```

Comment:
You should use a variable for your credentials in Postman so they're not checked into a public repository.

Reply:
ok will change